iOS/baseball-game/baseball-game/Selection/SelectionViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SelectionViewController.swift
+//  baseball-game
+//
+//  Created by Lia on 2021/05/03.
+//
+
+import UIKit
+
+class SelectionViewController: UIViewController {
+
+    @IBOutlet weak var backgroundView: UIView!
+    @IBOutlet weak var gameListTableView: UITableView!
+    
+    private let gradientLayer: CAGradientLayer = CAGradientLayer()
+    
+    var viewModel = SelectViewModel()
+    private var dataSource: UITableViewDiffableDataSource<Section, Game>!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setBackground()
+        
+        self.viewModel.request()
+        self.registerNib()
+        self.configureDataSource()
+        self.createSnapshot()
+    }
+
+}
+
+
+extension SelectionViewController {
+    enum Section {
+        case main
+    }
+    
+    private func registerNib() {
+        self.gameListTableView.register(GameCell.nib, forCellReuseIdentifier: GameCell.reuseIdentifier)
+    }
+    
+    private func configureDataSource() {
+        self.dataSource = UITableViewDiffableDataSource.init(tableView: self.gameListTableView) { (tableView, indexPath, game) -> UITableViewCell in
+            
+            let cell = self.gameListTableView.dequeueReusableCell(withIdentifier: GameCell.reuseIdentifier) as! GameCell
+            cell.fill(self.viewModel, state: game)
+            
+            return cell
+        }
+    }
+    
+    private func createSnapshot() {
+        var snapshot = NSDiffableDataSourceSnapshot<Section, Game>()
+        
+        snapshot.appendSections([.main])
+        viewModel.didFetchData { games in
+            snapshot.appendItems(games)
+            self.dataSource.apply(snapshot)
+        }
+    }
+}
+
+
+extension SelectionViewController {
+
+    private func setBackground() {
```

Comment:
> 네이밍

```suggestion
    private func configureBackgroundUI() {
```

=> set이라는 설정 함수가 존재하고 background도 여러 의미를 담고 있어 위의 메소드 이름이 더 적절할 것 같은데 어떠신가요?
=> 하나의 개념에는 하나의 단어를 사용해야 합니다. 현재 사용하는 `set`과 `configure`라는 메소드가 역할이 겹치네요. 또 이 `set`은 설정 함수인 `set` 가 겹치고요. 
=> 클린코드 2장 `의미있는이름 - 한 개념에 한 단어를 사용하라` 항목에도 똑같은 내용이 있으니 여유 있으실 때 읽는 것 추천드려요!

Reply:
넵 말씀해주신대로 `set`과 `configure`을 혼용해서 쓰는 것보다 한 가지로 통일해서 쓰는 것이 훨씬 바람직해 보입니다.
그리고 `background` 도 background music 처럼 다양하게 쓰일 수 있어, 화면으로 한정하는 것이 좋을 것 같습니다. (말씀해주신대로!)
추천해주신 함수명으로 바꾸도록 하겠습니다!